wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_messages.scss

```diff
@@ -44,12 +44,20 @@
     }
 
     .warning {
-        background-color: $color-orange;
+        background-color: $color-button-warning;
+        color: $color-button-warning-text;
 
         &:before {
             font-family: wagtail;
             content: '!';
         }
+
+        .avatar {
+            &:before {
+                color: $color-button-warning-text;
+                border: 1px solid $color-button-warning-text;
+            }
+        }
```

Comment:
I don't understand those changes to the avatar, what is the issue accessibility-wise?